src/app.js

```diff
@@ -214,26 +214,28 @@ function openOrFocusTab(url) {
   if (appState.workTab === null) {
     app.makeNewWorkTab(url);
   } else {
-    app.refreshTabInfo();
+    app.checkAndOpenTab(url);
   }
 }
 
-// Make sure the work tab is open and in focus
-function refreshTabInfo() {
+// check to see if tab is valid to decide to open new tab or not
+
+function checkAndOpenTab(url) {
   chrome.tabs.get(appState.workTab.id, tab => {
     if (chrome.runtime.lastError) {
       appState.workTab = null;
+      app.makeNewWorkTab(url);
     } else {
       appState.workTab = tab;
-
-      // force selection
-      if (!appState.workTab.selected) {
-        chrome.tabs.update(appState.workTab.id, {selected: true});
+      const re = /tester\.rainforestqa\.com\/tester\//;  // test tab if worker is still working on the job
+      if (!re.test(tab.url)) {
+        app.makeNewWorkTab(url);
```

Comment:
I'm not sure why this is here - if the tester is still working on the job, then it should just select it, not open a new tab?